lib/job_aggregator.rb

```diff
@@ -2,16 +2,29 @@ class JobAggregator
   def initialize
     @ignore_link_text ||= []
     @url ||= ''
+    @num_pages = ARGV[0] ? ARGV[0].to_i : 1
```

Comment:
this may need to be give in a different way, considering the script lets you do multiple aggregators with a single execution; they won't all have the same number of pages:
https://github.com/michaelporter/job_scour/blob/master/jobs.rb#L16